build_tools/cmake/build_and_test_tsan.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Build the IREE project with CMake. Designed for CI, but can be run manually.
+# This is equivalent to the build_tools/cmake/rebuild.sh script except it
+# first deletes the build directory to deal with any faulty caching.
+
+set -x
+set -e
+
+# IREE_BYTECODE_MODULE_FORCE_SYSTEM_DYLIB_LINKER=ON because TSan doesn't work
+# with embedded linker.
+# IREE_BUILD_SAMPLES=OFF because samples assume embedded linker.
+./build_tools/cmake/clean_build.sh \
```

Comment:
I think you want to just invoke CMake directly. That script is doing some weird implicit stuff...